### PR TITLE
Added all NFRs for Operational and Environmental

### PR DESCRIPTION
**Pull request by RafeedIqbal**
All NFRs added for subsections in section 13. Not sure if there would be any requirements for productization when handing over to the city. Let me know and I will add them.

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
...
+\begin{enumerate} [{OE-IAS}1.]
+  \item Application should integrate with existing SharePoint repositories.
+\end{enumerate}
 \subsection{Productization Requirements}
-\lips
+N/A
 \subsection{Release Requirements}
-\lips
+\begin{enumerate} [{OE-REL}1.]
+  \item A change-log should be generated with every release documenting changes in
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:190]: This section should define what is considered to be a release. Is it every commit, every deliverable, every revision, every deployment? Also, I think changelog is just one word, no need for the hyphen

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
...
 \subsection{Wider Environment Requirements}
-\lips
+\begin{enumerate} [{OE-WE}1.]
+  \item Application should be functional on Mobile and Desktop web browser environments.
+  \item Application should be able to run on any device with a modern web browser.
+\end{enumerate}
 \subsection{Requirements for Interfacing with Adjacent Systems}
-\lips
+\begin{enumerate} [{OE-IAS}1.]
+  \item Application should integrate with existing SharePoint repositories.
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:184]: should also describe infor and MySDS integrations

File: docs/SRS-Volere/SRS.tex [hunk 3]
```diff
@@ -171,15 +171,27 @@ \subsection{Longevity Requirements}
...
 \subsection{Expected Physical Environment}
-\lips
+\begin{enumerate} [{OE-PE}1.]
+  \item Application should be functional in City of Hamilton, Water Division sites and offices.
+\end{enumerate}
 \subsection{Wider Environment Requirements}
-\lips
+\begin{enumerate} [{OE-WE}1.]
+  \item Application should be functional on Mobile and Desktop web browser environments.
+  \item Application should be able to run on any device with a modern web browser.
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:180]: scope this down to just Chrome and Microsoft Edge for now, and if the city wants other browsers we'll ask them.

File: docs/SRS-Volere/SRS.pdf
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.pdf:1]: Same thing here with the binary file

File: docs/SRS-Volere/SRS.tex (hunk 1 above)
Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex]: Added definition, fixed spelling.

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex:491]: Added requirements for EAM and MySDS integration.

File: docs/SRS-Volere/SRS.tex (hunk 3 above)
Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex]: Scoped it down to Chrome, Edge and Mobile Browsers.

File: docs/SRS-Volere/SRS.pdf
Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.pdf:1]: Merged main into branch to revert the PDF.

**Review by kyle4908 — APPROVED** [review]

**Pull request merged**